src/scene/scene.js

```diff
@@ -103,9 +106,11 @@ import { StandardMaterial } from './materials/standard-material.js';
  * child to the Application root entity.
  */
 class Scene extends EventHandler {
-    constructor() {
+    constructor(graphicsDevice) {
         super();
 
+        this.device = graphicsDevice || getApplication().graphicsDevice;
```

Comment:
Presumably we want to deprecate `scene.defaultMaterial` in favour of using 
`DefaultMaterial.get()` directly? If that's the case, then we probably don't want to store device here (and change the constructor), but instead use `getApplication().graphicsDevice()` directly in the getter and flag the function as deprecated. And we want to change engine code to use `DefaultMaterial.get()`.